README.md

```diff
@@ -408,6 +408,41 @@ static:
         - Minimum
         period: 60
         length: 300
+  - namespace: AWS/SES
```

Comment:
Rather than this full example, maybe just add SES to the list of supported services at the top of the file.

Reply:
Ignore this comment, the service is already there. So this is just a fix to add the `ResourceFilters`?

Reply:
Does the `ResourceFilter` affect this? 
https://github.com/nerdswords/yet-another-cloudwatch-exporter/issues/738